tools/test/examples/examples.json

```diff
@@ -0,0 +1,8 @@
+{
+    "https://developer.mbed.org/teams/mbed/code/mbed_blinky" : ["K64F"],
+    "https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-Beacon" : ["NRF51_DK"],
```

Comment:
Ideally, we wouldn't maintain the list of BLE targets, we would just build for all supported targets that have the BLE feature turned on.